TECH_SPEC.md

```diff
@@ -56,7 +56,7 @@ Users can redeem their shares at any point in time if there is liquidity availab
 
 Optionally, a user can specify a list of strategies to withdraw from. If a list of strategies is passed, the vault will try to withdraw from them.
 
-If a user passed array is not defined. The redeem function will check if there is a queue_manager set to get a valid withdraw queue from. If neither happens the vault will check if there are enough idle funds to serve the request. If there are not enough, it will revert. 
+If a user passed array is not defined. The redeem function will use the default_queue.. 
```

Comment:
"If a user passed array is not defined, the redeem function will use the default_queue."